app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -5,42 +5,43 @@ class AddressSearchController < ApplicationController
 
       check_or_render_not_found -> { IdentityConfig.store.arcgis_search_enabled }
 
+      rescue_from Faraday::ConnectionFailed, Faraday::TimeoutError, with: :report_errors
+
       def index
         response = addresses(params[:address])
+
         render(**response)
       end
 
       protected
 
       def addresses(search_term)
         addresses = geocoder.find_address_candidates(SingleLine: search_term).slice(0, 1)
+
         { json: addresses, status: :ok }
-      rescue Faraday::ConnectionFailed => err
-        analytics.idv_arcgis_request_failure(
-          api_status_code: 422,
-          exception_class: err.class,
-          exception_message: err.message,
-          response_body_present: err.respond_to?(:response_body) && err.response_body.present?,
-          response_body: err.respond_to?(:response_body) && err.response_body,
-          response_status_code: err.respond_to?(:response_status) && err.response_status,
-        )
-        { json: [], status: :unprocessable_entity }
-      rescue Faraday::TimeoutError => err
-        analytics.idv_arcgis_request_failure(
-          api_status_code: 422,
-          exception_class: err.class,
-          exception_message: err.message,
-          response_body_present: err.respond_to?(:response_body) && err.response_body.present?,
-          response_body: err.respond_to?(:response_body) && err.response_body,
-          response_status_code: err.respond_to?(:response_status) && err.response_status,
-        )
-        { json: [], status: :unprocessable_entity }
       end
 
       def geocoder
         @geocoder ||= IdentityConfig.store.arcgis_mock_fallback ?
           ArcgisApi::Mock::Geocoder.new : ArcgisApi::Geocoder.new
       end
+
```

Comment:
my[ pr refactors this a bit](https://github.com/18F/identity-idp/pull/7763/files). i'm currently working through a merge conflict with my pr and the usps controller error handling refactor. Can we pause a moment before merging this so I can take a look and figure out how this works with my changes?